packages/node-opcua-client/source/alarms_and_conditions/client_alarm_tools.ts

```diff
@@ -45,6 +45,18 @@ export async function uninstallAlarmMonitoring(session: ClientSession): Promise<
     return;
 }
 
+/**
+ * Checks if all fields in the `ClientSessionPriv` interface are valid.
+ * @param session - The `ClientSessionPriv` object to check.
+ * @returns `true` if all fields are valid, `false` otherwise.
+ */
+export function areClientSessionPrivFieldsValid(session: ClientSessionPriv): boolean {
```

Comment:
This should not be a exported function. 

The function is probably not needed either as it could be incorporated easily in line 35